server/neptune/workflows/activities/terraform.go

```diff
@@ -219,21 +235,25 @@ func (t *terraformActivities) TerraformPlan(ctx context.Context, request Terrafo
 		},
 	}
 	args = append(args, request.Args...)
-	var flags []terraform.Flag
+	var flags []command.Flag
 
 	if request.PlanMode != nil {
 		flags = append(flags, request.PlanMode.ToFlag())
 	}
 
 	envs, err := getEnvs(request.DynamicEnvs)
-
 	if err != nil {
 		return TerraformPlanResponse{}, err
 	}
 
-	planRequest := &terraform.RunCommandRequest{
+	envs[TFInAutomation] = TFInAutomationVal
+	envs[AtlantisTerraformVersion] = tfVersion.String()
+	envs[Dir] = request.Path
+	envs[TFPluginCacheDir] = t.CacheDir
```

Comment:
It kinda sucks that we're doing the same thing 3 different times. It's pretty error prone. Can you like dedup this with the other functions? or keep this in the getEnvs function?

Reply:
`getEnvs` is also used by ExecuteCommand, but deduping with a helper fxn instead